routes/web.php

```diff
@@ -15,7 +15,4 @@
     return view('welcome');
 });
 
-Route::get('/get_role/{email}', 'UserRoleController@getRole');
-
-/** Allows laravel to be able to serve/host any vue page */
-Route::get('/{any}', 'SpaController@index')->where('any', '.*');
+Route::post('loginVerification', 'LoginController@authenticateUser');
```

Comment:
you need to add back the routes on line 18 and 21. `SpaController` is to return vue pages and `UserRoleController` is so front-end can hit that api